URBNSwiftCarousel/Classes/URBNSwiftCarouselProtocols.swift

```diff
@@ -0,0 +1,49 @@
+//
+//  URBNSwiftCarouselProtocols.swift
```

Comment:
verdict is there must be a compatibility layer, but we should not compromise the design of the core for that requirement.